joplin/locations/models.py

```diff
@@ -10,16 +13,47 @@
 )
 from base.models.translated_image import TranslatedImage
 
-# The abstract model for related links, complete with panels
-from wagtail.search import index
-from wagtail.admin.edit_handlers import FieldPanel, PageChooserPanel, HelpPanel
-from wagtail.images.edit_handlers import ImageChooserPanel
+
 from base.models import JanisBasePage
-from base.models.widgets import countMe, countMeTextArea, AUTHOR_LIMITS
-from modelcluster.models import ClusterableModel
+from base.models.widgets import countMe
 from base.models.constants import DEFAULT_MAX_LENGTH
 
 
+def add_hours_by_day_and_exceptions(model):
```

Comment:
I just moved this function to the top of the file to make it more obvious that something programmatic is happening here.